.storybook/main.js

```diff
@@ -11,6 +11,13 @@ module.exports = {
 		const rules = config.module.rules;
 		const { extensions } = config.resolve;
 
+		// Mock JSDOM for storybook - it relies on native node.js packages
+		// Allows us to use enhancers in stories for better testing of compoenents & full articles
+		config.resolve.alias.jsdom$ = path.resolve(
+			__dirname,
+			'./mocks/jsdom.js',
```

Comment:
might just be me, but the tab size looks huge in this file

Reply:
Yea it definitely looks massive but renders normally in vscode and has the same tabsize as all the other files (4 spaces) so very confusing...